test/com/facebook/buck/android/AndroidLibraryIntegrationTest.java

```diff
@@ -74,6 +74,16 @@ public void testAndroidKotlinLibraryCompilation() throws Exception {
     result.assertSuccess();
   }
 
+  @Test
+  public void testAndroidKotlinLibraryMixedSourcesCompilation() throws Exception {
+    AssumeAndroidPlatform.assumeSdkIsAvailable();
+    KotlinTestAssumptions.assumeCompilerAvailable(workspace.asCell().getBuckConfig());
+    KotlinTestAssumptions.assumeCompilerSupportsMixedSources();
```

Comment:
Can we just use junit's `@Ignore` annotation and link it with a github issue for bumping the compiler? 
Something like `@Ingore("https....github/issues/1234")`